packages/bpk-component-price/src/BpkPrice.js

```diff
@@ -63,9 +62,15 @@ const BpkPrice = (props: Props) => {
           {subtitle}
         </BpkText>
       )}
-      <div className={isSmall && getClassName('bpk-price__titleColumn')}>
+      <div
+        className={isAlighRight && getClassName('bpk-price__columnContainer')}
```

Comment:
Fixing spelling and SCSS suggestion made here: 
```suggestion
        className={isAlignRight && getClassName('bpk-price__column-container')}
```